packages/block-library/src/query/edit/index.js

```diff
@@ -37,7 +41,7 @@ export default function QueryEdit( {
 			<QueryProvider>
 				<InnerBlocks template={ TEMPLATE } />
 			</QueryProvider>
-		</>
+		</Block.div>
```

Comment:
I'm not familiar with this block and I don't see that toolbar, but I wonder if it makes sense to move the `BlockControls` outside the wrapper.

Reply:
Good catch!  It should probably not be wrapping the `BlockControls` as well.

Reply:
>  I don't see that toolbar

I moved `BlockControls` outside of the wrapper, but I had the toolbar working the same in either case.  Is it better for you now?